src/Blocks/BlockTorch.h

```diff
@@ -214,3 +219,41 @@ class cBlockTorchHandler:
 
 
 
+
+class cBlockTorchHandler final :
+	public cClearMetaOnDrop<cBlockStaveHandler>
+{
+	using Super = cClearMetaOnDrop<cBlockStaveHandler>;
+
+public:
+
+	using Super::Super;
+};
+
+
+
+
+
+class cBlockRedstoneTorchHandler final :
```

Comment:
Why are you moving this around?

Reply:
Consistency with e.g. BlockFluid since it's a super simple inherit of a parent, would you prefer another way?